recipe/meta.yaml

```
@@ -55,7 +55,7 @@ requirements:
   build:
      # ensure that all variants of cuda_compiler_version are tested
     - {{ compiler('cuda') }}              # [cuda_compiler_version != "None"]
-    - {{ compiler('c') }}                 # [cuda_compiler_version != "None"]
+    - {{ c_compiler }}_{{ target_platform }} {{ c_compiler_version }} *_4             # [cuda_compiler_version != "None"]
```

Guessing we would want something like this (as Windows wouldn't need this pin)

```suggestion
    - {{ c_compiler }}_{{ target_platform }} {{ c_compiler_version }} *_4             # [linux and cuda_compiler_version != "None"]
    - {{ compiler('c') }}                                                                # [(not linux) and cuda_compiler_version != "None"]
```

---

Yes but this is just for testing. If this works, we'll want to use a repodata patch.